icubam/messaging/sms_sender_test.py

```diff
@@ -21,6 +21,13 @@ def test_from_string(self):
       sms_sender.get(self.config, 'Fake'), sms_sender.FakeSender)
     self.assertIsInstance(
       sms_sender.get(self.config, 'fake'), sms_sender.FakeSender)
+    with self.assertRaises(ValueError):
```

Comment:
might be good to use `assertRaisesRegex` with part of the error message, to be more specific.

Reply:
Good idea !